examples/create-default-webpack-mf/README.md

```diff
@@ -8,12 +8,13 @@ Run the following commands in the root directory.
 yarn
 yarn start
 ```
-
 Both `app1` and `app2` are independently deployed apps:
 
 - `app1`: http://localhost:3001
 - `app2`: http://localhost:3002
 
+To be able to use this example, build app2. Then find it in https://app.zephyr-cloud.io and set link to 'remoteEntry.js' file. Link should be set in variable app2Url (/app1/src/index.js).
```

Comment:
This is only needed when running locally right?
Zephyr delegate is responsible for resolving this URL because it replaces it in build time.

Reply:
hmm, i am not sure. I used prod env. and i remember it doesn't work anyway without app2Url. I will test one more time and will answer definitely.

Reply:
yes, you are right if such remote is already on zephyr, this variable can be empty, build is success